modules/500-dashboard/images/web/patches/logout_button.js

```diff
@@ -0,0 +1,53 @@
+
+function setupLogoutButton() {
+  let flag = false;
+
+  document.querySelectorAll("div").forEach(function (node) {
+    if (node.innerHTML.startsWith("Logged in with auth header") === true) {
+      let r = new XMLHttpRequest();
```

Comment:
Does not necessarily needs to be addressed, but XHR in 2024? Would be much simpler to use Fetch API.